backend/README.md

```diff
@@ -26,5 +26,12 @@ python3 -m pytest
 From `backend` run:
 
 ```bash
-python server/app.py
-```
\ No newline at end of file
+python3 server/app.py
```

Comment:
py server/app.py

Reply:
It doesn't work with command 'py server/app.py' because we use Python 3.
But we can also use command 'python server/app.py'

Reply:
Fixed README, it works with just 'python'